config/initializers/assets.rb

```diff
@@ -25,7 +25,7 @@
   files.delete('.')
   files.delete('..')
 
-  Rails.application.config.assets.paths << directory
+  Rails.application.config.assets.paths << directory.to_s
```

Comment:
anders is het een array oid? Nice iig

Reply:
Ja anders is het een of ander ruby Directory object oid. Ik bekeek iig de assets.paths in de console en het waren allemaal strings behalve dit ene Directory/Path object.